frontend/.eslintrc

```diff
@@ -101,6 +100,8 @@
     ],
     // Disable a bunch of AirBNB rules we're currently in violation of.
     // TODO: For each one, either fix and reenable, or provide a justification.
+    "@typescript-eslint/no-shadow": "off",
+    "import/no-named-as-default-member": "off",
```

Comment:
Can you add comment above each of these explaining why it's disabled (and if appropriate, what needs to happen if we want to re-enable it)?